packages/tocco-ui/src/Typography/Misc.js

```diff
@@ -74,6 +75,13 @@ const Kbd = props =>
     title={props.breakWords ? undefined : getTextOfChildren(props.children)}
   >{props.children}</StyledKbd>
 
+const Label = props =>
+  <StyledLabel
+    breakWords={props.breakWords}
+    title={props.breakWords ? undefined : getTextOfChildren(props.children)}
+    htmlFor={props.for}
+  >{props.children}</StyledLabel>
```

Comment:
new line for </StyledLabel>

Reply:
Either there's some part of this comment missing or I just don't understand =) I copied the layout from the other elements in this file, lemme know if there's something that should change

Reply:
sure something's missing here. anyhow i thought of rewriting it to this:

```
const Label = props =>
  <StyledLabel
    breakWords={props.breakWords}
    title={props.breakWords ? undefined : getTextOfChildren(props.children)}
    htmlFor={props.for}
  >
      {props.children}
  </StyledLabel>

```

but you have a point there with the consistency in the file. better leave it.